lib/routes/jimmyspa/books.ts

```diff
@@ -0,0 +1,103 @@
+import { Route, ViewType } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { art } from '@/utils/render';
+import { getCurrentPath } from '@/utils/helpers';
+const __dirname = getCurrentPath(import.meta.url);
+import path from 'node:path';
+
+export const route: Route = {
+    path: '/books/:language',
+    categories: ['design'],
+    view: ViewType.Articles,
+    example: '/jimmyspa/books/tw',
+    parameters: {
+        language: {
+            description: '语言',
+            options: [
+                { value: 'tw', label: '臺灣正體' },
+                { value: 'en', label: 'English' },
+                { value: 'jp', label: '日本語' },
+            ],
+        },
+    },
+    radar: [
+        {
+            source: ['www.jimmyspa.com/:language/Books'],
+        },
+    ],
+    name: 'Books',
+    description: `
+| language | Description |
+| ---   | ---   |
+| tw | 臺灣正體 |
+| en | English |
+| jp | 日本語 |
+    `,
+    maintainers: ['FYLSen'],
+    handler,
+};
+
+async function handler(ctx) {
+    const language = ctx.req.param('language');
+    const rootUrl = 'https://www.jimmyspa.com';
+
+    const currentUrl = new URL(`/${language}/Books/Ajax/changeList?year=&keyword=&categoryId=0&page=1`, rootUrl).href;
+
+    const responseData = await got(currentUrl);
+    const $ = load(responseData.data.view);
+
+    const items = $('ul#appendWork li.work_block')
+        .toArray()
+        .map(async (item) => {
+            const $$ = load(item);
+            const title = $$('p.tit').text();
+            const imagesrc = $$('div.work_img img').prop('src') || '';
+            const image = imagesrc ? rootUrl + imagesrc : '';
+            const link = $$('li.work_block').prop('data-route');
+            const contData = await got(link);
```

Comment:
Please [cache](https://docs.rsshub.app/joinus/advanced/use-cache) the content since it's less likely to be changed.